brion/target.cpp

```diff
@@ -147,11 +147,12 @@ const Strings& Target::get( const std::string& name ) const
 
 namespace
 {
-void _parse( const Targets& targets, const std::string& name, GIDSet& gids )
+void _parse( const Targets& targets, const std::string& name, GIDSet& gids, bool& found )
```

Comment:
Have you considered making _parse throw? Maybe the code is simpler, overall if both Target::parse and Target::get throw.